setup.py

```diff
@@ -67,7 +76,7 @@
             'pyobjc-core>=3.0.3',
             'pyobjc-framework-Cocoa>=3.0.3',
             'pyobjc-framework-Quartz>=3.0.3',
-            'wxPython>=2.8',
+            'wxPython>=3.0',
```

Comment:
If we increase the version requirements here (which I'm fine with), than we should do it in `plover/main.py` too, no?